### PR TITLE
Added API Name as configuration option

### PR DESCRIPTION
**Pull request by aaron-prindle**
Fixes: #1080

### CODE REVIEW

File: cmd/localkube/cmd/options.go [hunk 1]
```diff
@@ -59,6 +60,8 @@ func AddFlags(s *localkube.LocalkubeServer) {
 	flag.IntVar(&s.APIServerPort, "apiserver-port", s.APIServerPort, "The port the apiserver will listen securely on")
 	flag.IPVar(&s.APIServerInsecureAddress, "apiserver-insecure-address", s.APIServerInsecureAddress, "The address the apiserver will listen insecurely on")
 	flag.IntVar(&s.APIServerInsecurePort, "apiserver-insecure-port", s.APIServerInsecurePort, "The port the apiserver will listen insecurely on")
+	flag.StringVar(&s.APIServerName, "apiserver-name", s.APIServerName, "The apiserver name for localkube/kubernetes")
```
Comment (dims) [inline cmd/localkube/cmd/options.go]: Can we please add something to the effect that this name is used in the generated certificate? Will be helpful to someone just running help commands.

**Comment by aaron-prindle** [comment]
I have updated the help text to read: "The API server name which is used in the ge\
   nerated certificate for localkube/kubernetes.  This can be used if you want to make the API server available from outside the machine""

**Comment by dims** [comment]
Looks good to me @aaron-prindle thanks!

**Comment by aaron-prindle** [comment]
@r2d4 no, that doesn't need to be changed

**Pull request merged**